.github/policies/resourceManagement.yml

```diff
@@ -170,5 +170,27 @@ configuration:
         
         - [ ] Complete [localizability testing](https://github.com/NuGet/NuGet.Client/blob/dev/docs/ui-guidelines.md#localizability-testing).'
       description: 'Remind PR author to test accessibility and localizability'
+    - if:
+      - payloadType: Pull_Request
+      - labelAdded:
+          label: breaking-change

```

Comment:
question: Continuing the conversation from the previous PR:

The Category:BreakingChange label is in the Home repo and applies to issues.

I was thinking whether we key off the issue breaking label, but sounds there's a broader thing going on where the label is expected on the PRs I guess, is that correct?

If that's the case, then we can just keep the label and align our workflow to that. 

`breaking-change` applies to end user .NET SDK breaking changes correct?
I think we'd probably just define that in the label description.

Reply:
It's expected in both the issues and PRs.  I already renamed Category:BreakingChange to breaking-change. I will add a description to the label per your suggestion.

Reply:
Okay, actually, all these comments are only relevant to https://github.com/NuGet/Home/pull/14211. Could we copy/paste there? 

In NuGet.Client, the breaking-change label is brand-new, so it doesn't have any impact on existing processes / tooling.